Google/index.html

```diff
@@ -0,0 +1,157 @@
+<html>
+  <head>
+    <style>
+    header {height: 100%;}
+    body {
+        margin: 0;
+        padding: 0;
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        }
+    * {font-family: sans-serif;}
+    nav {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding-left: 10px;
+        padding-right: 10px;
+        }
+    .nav-right {
+        display: flex;
+        align-items: center;
+        }
+    nav a, nav img {
+        margin-right: 15px;
+        }
+    a {
```

Comment:
You styled all your links to have black color. But french-translation link should be regular colored link.
Be careful when you create general style rule that will affect all elements.
Better to create classes for specific cases, class="nav-link" for navigation links, class="footer-link"
class="translation-link".
Exercise example style rule is  `nav a {color: #333}`
Just something for you to consider.